core/tests/karma.conf.ts

```diff
@@ -104,16 +104,15 @@ module.exports = function(config) {
     singleRun: true,
     customLaunchers: {
       Chrome_Travis: {
-        // Karma can only connect to ChromeHeadless when inside Docker.
-        base: isDocker ? 'ChromeHeadless' : 'Chrome',
+        base: 'ChromeHeadless',
```

Comment:
Out of curiosity, was there a reason why we didn't run our tests in a headless browser?

Reply:
I think we just couldn't get it to work -- at the time we were playing with PhantomJS which has since been deprecated IIRC. 

Could you just do a test run to ensure that, if there's some sort of error, the tests do actually fail with this config?

Reply:
Made 0cd80ad to trigger an error on purpose.

Reply:
^ it does error locally though, I'm doing another test for Github Actions.

Reply:
Yup, that commit does fail on github actions.